pyvista/core/dataset.py

```diff
@@ -2155,16 +2178,26 @@ def find_closest_cell(self, point: Union[int, np.ndarray]) -> Union[int, np.ndar
         locator.BuildLocator()
 
         cell = _vtk.vtkGenericCell()
-        closest_point = [0, 0, 0]
-        cellId = _vtk.mutable(0)
-        subld = _vtk.mutable(0)
-        dist2 = _vtk.mutable(0.0)
 
         closest_cells = []
+        closest_points = []
+
         for node in point:
+            closest_point = [0, 0, 0]
+            cellId = _vtk.mutable(0)
+            subld = _vtk.mutable(0)
+            dist2 = _vtk.mutable(0.0)
+
             locator.FindClosestPoint(node, closest_point, cell, cellId, subld, dist2)
```

Comment:
I know this was not added here, but I just noticed the `camelCase`. I'd prefer to switch to `cell_id` anyway, but I noticed that `subId` has become `subld` which is another excellent reason why we should call it `sub_id` :)